### ISSUE TITLE
Recipe #269: Embedded or Referenced Annotations

### ISSUE THREAD

**Issue by glenrobson** [IIIF/trc#81]
## Links

 * Original Issue: https://github.com/IIIF/cookbook-recipes/issues/269
 * Pull Request: https://github.com/IIIF/cookbook-recipes/pull/274/
 * Preview: https://preview.iiif.io/cookbook/0269-embedded-or-referenced-annotations/recipe/0269-embedded-or-referenced-annotations/

## Background and Summary

We have submitted a number of annotation recipes to the TRC and these have been building up in complexity. So far they have all been annotations that have been embedded in the Manifest as this has been the easiest to demonstrate the particular feature the recipe is highlighting. In practice there tends to be many annotations associated with an item and this means it is usually too big to embed in the Manifest so the alternative is to reference them as demonstrated in this recipe.  

## Voting and changes

We welcome comments on the recipe and as well as voting +1, confused face or -1 feel free to add comments to this issue. If this issue is approved then the author will take account of the comments before we merge the branch in to the master cookbook branch.

If the recipe is rejected by the TRC then we will make the changes requested and resubmit it to a future TRC meeting. If you feel that your comments are substantial enough that the recipe should be looked at again by the TRC after the changes have been made please vote -1 (thumbs down). A confused face is treated as abstaining. 

Changes to the recipe will only be made after the TRC voting process has concluded.

**Comment by regisrob** [comment]
Modulo the wrong [View in Mirador](https://projectmirador.org/embed/?iiif-content=/cookbook/0269-embedded-or-referenced-annotations/recipe/0269-embedded-or-referenced-annotations/manifest.json) link, but this is a website issue

**Comment by glenrobson** [comment]
Yes the link to the manifest is relative rather than containing the full domain name and path to the manifest and this is an issue with the website move. Its fixed in the main branch but will need to updated into this branch after the vote finishes tomorrow.

**Comment by glenrobson** [comment]
### Issue 81 (Recipe #269: Embedded or Referenced Annotations)
  +1: 17 [brndgtl cubap emulatingkat glenrobson hadro irv jcreel jtweed julsraemy kirschbombe ksclarke mixterj mposton-folger nfreire regisrob tpendragon triplingual]
   0: 0 []
  -1: 0 []
  Not TRC: 0 []
  Ineligible: 1 [cjnishioka]

### Result: 17 / 17 = 1.00
Super majority is in favor, issue is approved
